draft-ietf-tls-tls13.md

```diff
@@ -3413,11 +3425,12 @@ ticket_extensions
 : A placeholder for extensions in the ticket. Clients MUST ignore
   unrecognized extensions.
 
-ticket
-: The value of the ticket to be used as the PSK identifier.
-The ticket itself is an opaque label. It MAY either be a database
-lookup key or a self-encrypted and self-authenticated value. Section
-4 of {{RFC5077}} describes a recommended ticket construction mechanism.
+tickets
+: One or more ticket values, each of which is to be used as the PSK
```

Comment:
Maybe explicitly say that the multiple values are just concatenated one after each other with no additional framing other than that from the opaque Ticket<>?  I don't see how else someone would interpret this, but just in case...